src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views.stories.tsx

```diff
@@ -45,8 +38,4 @@ PureComponent.argTypes = {
     control: 'boolean',
     defaultValue: true,
   },
-  dataViewsDocLink: {
```

Comment:
Why this change? `dataViewDocLinks` is an optional property, and `PureComponent` can be rendered with our without it and it will look differently in either case, which is what this control was showing.

Reply:
Mistake on my part, I'll put this back.